rkt/run.go

```diff
@@ -328,6 +340,22 @@ func (pl *portList) Type() string {
 	return "portList"
 }
 
+// flagStringList implements the flag.Value interface to contain a set of strings
+type flagStringList []string
```

Comment:
However, `rkt/run.go` already has some flag types: `envMap`, `portList`. Should I still move it to `rkt.go` which does not have any flag type yet?